index.html

```diff
@@ -0,0 +1,212 @@
+                <h3 class="proj-head">Professional Art Printing Data</h3>
+                <p class="proj-para">
+               A daily selection of privately personalized reads; no accounts or
+               sign-ups required. has been the industry's standard dummy text ever
+               since the 1500s, when an unknown printer took a standard dummy text.
+                </p>
+               <ul class="flex project btn-flex">
+                  <li class="proj-btn">html</li>
+                  <li class="pproj-btn">bootstrap</li>
+                  <li class="proj-btn">Ruby</li>
+               </ul>
+              </div>
+              <a href="/" class="cursor project-link">See Project</a>
+             </li>
+             <li class="proj-con">
+              <div class="proj-txt">
+                <h3 class="proj-head">Professional Art Printing Data</h3>
+                <p class="proj-para">
+               A daily selection of privately personalized reads; no accounts or
+               sign-ups required. has been the industry's standard dummy text ever
+               since the 1500s, when an unknown printer took a standard dummy text.
+                </p>
+               <ul class="flex project btn-flex">
+                  <li class="proj-btn">html</li>
+                  <li class="pproj-btn">bootstrap</li>
+                  <li class="proj-btn">Ruby</li>
+               </ul>
+              </div>
+              <a href="/" class="cursor project-link">See Project</a>
+            </li>
+            <li class="proj-con">
+              <div class="proj-txt">
+                <h3 class="proj-head">Professional Art Printing Data</h3>
+                <p class="proj-para">
+               A daily selection of privately personalized reads; no accounts or
+               sign-ups required. has been the industry's standard dummy text ever
+               since the 1500s, when an unknown printer took a standard dummy text.
+                </p>
+               <ul class="flex project btn-flex">
+                  <li class="proj-btn">html</li>
+                  <li class="proj-btn">bootstrap</li>
+                  <li class="proj-btn">Ruby</li>
+               </ul>
+              </div>
+              <a href="/" class="cursor project-link">See Project</a>
+           </li>
+            </ul>
+          </section>
+          <section class="about-section">
+            <h2 class="greetings about">About me</h2>
+            <div class="about-container img2">
+              <p class=" intro">
+                Hello I'm a software developer! I can help you build a product,
+                feature or website. Look through some of my work and experience! If
+                you like what you see and have a project you need coded, dont hestiate
+                to contact me.
+              </p>
+              <a class="btn" href="#">Get My Resume</a>
+              <p class="underline"></p>
+            </div>
```

Comment:
This section is missing the proper background decorations and text colour please make the necessary adjustments.
Yours :
![Screenshot (65)](https://user-images.githubusercontent.com/52497246/121771749-3ff5a580-cb69-11eb-9a36-870397c50eb3.png)

Figma Design:
![Screenshot (66)](https://user-images.githubusercontent.com/52497246/121771777-66b3dc00-cb69-11eb-96fd-37c0e8deb579.png)

Reply:
Hello, Good morning. I have been on this project for the past two weeks. As the TSE for this project I believe it was your job to point me in the right direction from the very first review, you have failed to do that and I have gone ahead to make all the Changs you requested, now comes the second review asking me to do something totally different, I went ahead and did that, only for you to come back and demand I made changes you didn’t spot in the first review. This is Crass, and I’m left with just one review and you know what happens when I fail that, I will suggest you stick to a first reviewer's comments and go on from there, instead of driving me on a railroad with no end in sight. Why did you not spot these Changes in your first CR? I need Clarification and a better approach so I can submit this and move on to the next project.

Reply:
So sorry for not pointing out these changes earlier, it's either they weren't there or it skipped me @Kaneez98Fatima we Code reviewers are here to help you make professional projects with best practices, please 🙏 make the requested changes as mentioned above and the next reviewer would know to approve thanks.

Reply:
> So sorry for not pointing out these changes earlier, it's either they weren't there or it skipped me @Kaneez98Fatima we Code reviewers are here to help you make professional projects with best practices, please make the requested changes as mentioned above and the next reviewer would know to approve thanks.

It's fine @Davidosky007  I make changes in the project again to the best of my knowledge...Hope for the best :1st_place_medal: